src/Enhavo/Bundle/RevisionBundle/DependencyInjection/EnhavoRevisionExtension.php

```diff
@@ -23,7 +19,7 @@ public function load(array $configs, ContainerBuilder $container): void
 
         $config = $this->processConfiguration($configuration, $configs);
         $loader = new Loader\YamlFileLoader($container, new FileLocator(__DIR__.'/../Resources/config'));
-
+        $container->setParameter('enhavo_revision.doctrine_filter', $config['doctrine_filter']);
```

Comment:
Just save the enabled state in the configuration.

` $container->setParameter('enhavo_revision.doctrine_filter.enabled', $config['doctrine_filter']['enabled']);`